apis/python/src/tiledbsoma/io/conversions.py

```diff
@@ -50,7 +52,7 @@ def to_tiledb_supported_array_type(name: str, x: _MT) -> _MT:
     ):
         # mypy issues a spurious error here, but only when
         # _to_tiledb_supported_dtype is decorated with @typeguard_ignore???
```

Comment:
If the type-ignore is no longer needed, this comment can be removed.

Reply:
fixed